src/data_types/data_structures_comparison.md

```diff
@@ -0,0 +1,17 @@
+* Changeble: can add or remove items after the collection has been created
```

Comment:
Typo here. Should be `Changeable`. Although I personally think you should use `Mutable` instead.